pkg/operator/mon/mon.go

```diff
@@ -40,6 +40,10 @@ import (
 var logger = capnslog.NewPackageLogger("github.com/rook/rook", "op-mon")
 
 const (
+	// EndpointConfigMapName is the name of the configmap with mon endpoints
+	EndpointConfigMapName = "mon-config"
```

Comment:
Also, why are these variables capitalized? Are they referenced outside the package? It is because they are not consistent with the other variables in this struct.